### PR TITLE
[Azure Stack] Add TargetScaleUnit and TargetVolumeLabel parameters for disk migration job

### PR DESCRIPTION
**Pull request by zyxyoshine**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by zyxyoshine** [comment]
This is for Azure Stack API, @bganapa please help review.

File: specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/DiskMigrationJobs.json [hunk 1]
```diff
@@ -114,6 +114,12 @@
           {
             "$ref": "DiskMigrationJobs.json#/parameters/TargetShareParameter"
           },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/TargetScaleUnitParameter"
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/DiskMigrationJobs.json]: is this a bug fix? Can you please make another PR to master branch as well?

**Review by bganapa — APPROVED** [review]

File: specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/DiskMigrationJobs.json (hunk 1 above)
Comment (zyxyoshine) [reply specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/DiskMigrationJobs.json]: We have implemented this parameters for the API but did not add them in sepc, as the previous SDK does not need these parameters.
@bganapa If you are going to merge stackadmin to master, do I still need to create a new PR?
